test/brsTypes/components/RoSGNode.test.js

```diff
@@ -278,4 +278,335 @@ describe("RoSGNode", () => {
             });
         });
     });
+
+    describe("ifSGNodeChildren", () => {
+        let interpreter;
+
+        beforeEach(() => {
+            interpreter = new Interpreter();
+        });
+
+        describe("getchildcount", () => {
+            it("returns the number of children", () => {
+                let node = new RoSGNode([]);
+
+                let getChildCount = node.getMethod("getchildcount");
+                expect(getChildCount).toBeTruthy();
+
+                let result = getChildCount.call(interpreter);
+                expect(result).toEqual(new Int32(0));
+            });
+
+            //TODO: test when the node does have children
```

Comment:
What's this TODO for?

Reply:
oops this was left when we didn't have any other functions implemented.  I think other test cases have covered this well, will remove